.travis.yml

```diff
@@ -16,6 +16,7 @@ env:
     - TEST=flake8
     - TEST=snyk
     - TEST=docker
+    - TEST=unittests
```

Comment:
as discussed, how about calling that "integration_tests" in order not to mix them with the unit tests? the dojo.unittests are run under the helm context in travis script.sh main section
```
helm/defectdojo/templates/tests/unit-tests.yaml
      command: ['/entrypoint-unit-tests.sh']
        entrypoint-unit-tests.sh
          python manage.py makemigrations dojo
          python manage.py migrate
          exec python manage.py test dojo.unittests --keepdb
```
the "unittest-script.sh" script name should also be changed